app/views/sellers/orders/_order.html.erb

```diff
@@ -0,0 +1,38 @@
+<% order.order_items.each do |item| %>
+<tr class="hover:bg-blue-50 transition">
+  <td class="px-6 py-4 text-sm font-medium text-gray-800">
+    <%= order.id %>
+  </td>
+
+  <td class="px-6 py-4 text-sm text-center">
+    <%= number_to_currency(order.order_items.sum { |item| item.product.price * item.quantity }) %>
+  </td>
+  <td class="px-6 py-4 text-sm">
+    <%= form_with(model: order, url: sellers_order_path(order), method: :patch, 
+                  data: { turbo_frame: "order_#{order.id}" }, 
+                  class: "flex items-center space-x-2") do |form| %>
+      <%= form.select :status, 
+        options_for_select(
+          [['Confirmed', 2], ['Shipped', 3]], 
+          order.status
+        ), 
+        {}, 
+        class: "w-full border border-gray-300 rounded-md bg-white py-1 px-2 text-sm focus:outline-none focus:ring focus:ring-blue-500 focus:border-blue-500 text-gray-800" %>
+    <% end %>
+  </td>
+  <td class="px-6 py-4 text-sm text-center">
+    <%= order.user.name %>
+  </td>
+  <td class="px-6 py-4 text-center">
+    <%= form_with(model: order, url: sellers_order_path(order), method: :patch, 
```

Comment:
Why are you creating separate forms for the dropdown and submit button? This could have been accomplished with a single form.